scopes/ui-foundation/ui/ui.ui.runtime.tsx

```diff
@@ -67,20 +69,21 @@ export class UiUI {
       </Compose>
     );
 
-    renderContexts = await this.onAfterHydrate(renderContexts, lifecycleHooks, app);
+    renderContexts = await this.triggerBeforeHydrateHook(renderContexts, lifecycleHooks, app);
```

Comment:
extracted logic to their own method

Reply:
@KutnerUri This used to be called onAfterHydrate, and now it's beforeHydrate.
What is the correct one?